### PR TITLE
13356 Move more files out of cache

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-13356

### CODE REVIEW

**Comment by Sergio-Mira** [comment]
@adam-govan Could you have a look?

File: SwrveSDK/SDK/Track/Swrve.h [hunk 1]
```diff
@@ -216,35 +216,72 @@ typedef void (^SwrveResourcesUpdatedListener) ();
...
+
+/*! The event cache stores data that has not yet been sent to Swrve.
+ * If you plan to change this please contact the team at Swrve who will be happy to help you out.
  * This path should be located in app/Libraries/Caches/, as this is where Apple
  * recommend that persistent data should be stored. http://bit.ly/nCe9Zy
+ *
+ * This contains the pre iOS SDK 4.5.1 location for migration purposes.
  */
-@property (nonatomic, retain) NSString * eventCacheFile;
+@property (nonatomic, retain) NSString * eventCacheSecondaryFile;
```
Comment (dominicmarmionswrve) [inline SwrveSDK/SDK/Track/Swrve.h:229]: Can we come up with a better name for this instead of eventCacheSecondaryFile?

**Comment by dominicmarmionswrve** [comment]
Think we should really tidy this up and have a designated class for persisting long living data. Makes it easier to test, read, maintain and generally a better idea. Perhaps the swrve_install.txt should use this new class also.

File: SwrveSDK/SDK/Track/Swrve.h (hunk 1 above)
Comment (adam-govan) [inline SwrveSDK/SDK/Track/Swrve.h:229]: preMigrationCacheFile?

**Comment by Sergio-Mira** [comment]
🚀

File: SwrveSDK/SDK/Track/Swrve.h (hunk 1 above)
Comment (dominicmarmionswrve) [inline SwrveSDK/SDK/Track/Swrve.h:229]: Did we really go with eventCacheSecondaryFile?

**Pull request merged**